web/src/main/kotlin/com/example/ajaxproject/service/EmailSenderServiceImpl.kt

```diff
@@ -9,29 +9,37 @@ import org.slf4j.LoggerFactory
 import org.springframework.mail.javamail.JavaMailSender
 import org.springframework.mail.javamail.MimeMessageHelper
 import org.springframework.stereotype.Service
+import reactor.core.publisher.Mono
 
 @Service
 internal class EmailSenderServiceImpl(private val javaMailSender: JavaMailSender) : EmailSenderService {
 
-    override fun send(emailDTO: EmailDTO): SendEmailResponse {
-        return runCatching {
+    val emailRegex = "^[A-Za-z0-9+_.-]+@[A-Za-z0-9.-]+\$"
```

Comment:
Move this to companion object as a constant

Reply:
This discussion is silently resolved but the changes are not applied :(